pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -623,12 +651,12 @@ func selectVictimsOnNode(
 	// inter-pod affinity to one or more victims, but we have decided not to
 	// support this case for performance reasons. Having affinity to lower
 	// priority pods is not a recommended configuration anyway.
-	if status := fh.RunFilterPluginsWithNominatedPods(ctx, state, pod, nodeInfo); !status.IsSuccess() {
-		if status.Code() == framework.Error {
-			klog.Warningf("Encountered error while selecting victims on node %v: %v", nodeInfo.Node().Name, status.AsError())
+	if finalStatus = fh.RunFilterPluginsWithNominatedPods(ctx, state, pod, nodeInfo); !finalStatus.IsSuccess() {
+		if finalStatus.Code() == framework.Error {
```

Comment:
we probably don't need this, just return the status as is.

Reply:
Done